ncm-metaconfig/src/main/metaconfig/mailrc/tests/regexps/account_config

```diff
@@ -0,0 +1,9 @@
+account config test for mailrc
+---
+/etc/mail.rc
+multiline
+---
+^account gmail {$
+^set from=root@gmail.com$
+^set smtp=mailserver.gmail.com$
+^}$
```

Comment:
this very likely also requires unescaping

Reply:
escaping?